build: alpine qt from depends not configuring

Building master (619e930aa1860d92f8eebef3bc1bed8b37c606b8) in an [Alpine 3.13 container](https://github.com/fanquake/core-review/blob/master/docker/alpine.dockerfile):
```bash
make -C depends/ -j9
./autogen.sh
CONFIG_SITE=/bitcoin/depends/x86_64-pc-linux-musl/share/config.site ./configure
...
checking for QT_CORE... yes
checking for QT_GUI... yes
checking for QT_WIDGETS... yes
checking for QT_NETWORK... yes
checking for QT_TEST... yes
checking for QT_DBUS... yes
checking for static Qt... yes
checking for QT_ACCESSIBILITY... yes
checking for QT_DEVICEDISCOVERY... yes
checking for QT_EDID... yes
checking for QT_EVENTDISPATCHER... yes
checking for QT_FB... yes
checking for QT_FONTDATABASE... yes
checking for QT_THEME... yes
checking for QT_INPUT... yes
checking for QT_SERVICE... yes
checking for QT_XCBQPA... yes
checking for QMinimalIntegrationPlugin (-lqminimal)... yes
checking whether the linker accepts -lxcb-shm... yes
checking for QXcbIntegrationPlugin (-lqxcb)... no
configure: WARNING: QXcbIntegrationPlugin not found.; bitcoin-qt frontend will not be built
checking whether to build Bitcoin Core GUI... no
```

config.log excerpt:
```bash
configure:29416: checking for QMinimalIntegrationPlugin (-lqminimal)
configure:29434: g++ -m64 -std=c++17 -o conftest -fPIC -pipe -O2  -fno-extended-identifiers -DQT_NETWORK_LIB -I/bitcoin/depends/x86_64-pc-linux-musl/include/QtNetwork -DQT_CORE_LIB -I/bitcoin/depends/x86_64-pc-linux-musl/include/QtCore  -DQT_WIDGETS_LIB -I/bitcoin/depends/x86_64-pc-linux-musl/include/QtWidgets -I/bitcoin/depends/x86_64-pc-linux-musl/include/QtCore -DQT_GUI_LIB -I/bitcoin/depends/x86_64-pc-linux-musl/include/QtGui -DQT_CORE_LIB  -DQT_GUI_LIB -I/bitcoin/depends/x86_64-pc-linux-musl/include/QtGui -DQT_CORE_LIB -I/bitcoin/depends/x86_64-pc-linux-musl/include/QtCore  -DQT_CORE_LIB -I/bitcoin/depends/x86_64-pc-linux-musl/include/QtCore   -I/bitcoin/depends/x86_64-pc-linux-musl/include/  -DHAVE_BUILD_INFO -D__STDC_FORMAT_MACROS -L/bitcoin/depends/x86_64-pc-linux-musl/lib  conftest.cpp -lqminimal -L/bitcoin/depends/x86_64-pc-linux-musl/lib -lQt5XcbQpa /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5ServiceSupport.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5ThemeSupport.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5FontDatabaseSupport.a -lfontconfig -L/bitcoin/depends/x86_64-pc-linux-musl/lib -lfreetype /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Gui.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5EdidSupport.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5DBus.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread /bitcoin/depends/x86_64-pc-linux-musl/lib/libxcb-static.a -lxcb -lxkbcommon-x11 -lxkbcommon -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5Gui /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5Gui /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5ServiceSupport /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Gui.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5DBus.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5Gui /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5DBus /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5ThemeSupport /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Gui.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5DBus.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5Gui /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5FontDatabaseSupport /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Gui.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread -lfontconfig -L/bitcoin/depends/x86_64-pc-linux-musl/lib -lfreetype -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5Gui /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5EdidSupport /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a  -L/bitcoin/depends/x86_64-pc-linux-musl/lib -lQt5ServiceSupport /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Gui.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5DBus.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5Gui /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5DBus /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a  -L/bitcoin/depends/x86_64-pc-linux-musl/lib -lQt5XcbQpa /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5ServiceSupport.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5ThemeSupport.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5FontDatabaseSupport.a -lfontconfig -L/bitcoin/depends/x86_64-pc-linux-musl/lib -lfreetype /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Gui.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5EdidSupport.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5DBus.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread /bitcoin/depends/x86_64-pc-linux-musl/lib/libxcb-static.a -lxcb -lxkbcommon-x11 -lxkbcommon -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5Gui /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5Gui /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5ServiceSupport /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Gui.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5DBus.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5Gui /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5DBus /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5ThemeSupport /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Gui.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5DBus.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5Gui /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5FontDatabaseSupport /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Gui.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread -lfontconfig -L/bitcoin/depends/x86_64-pc-linux-musl/lib -lfreetype -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5Gui /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5EdidSupport /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a  -L/bitcoin/depends/x86_64-pc-linux-musl/lib -lQt5ThemeSupport /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Gui.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5DBus.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5Gui /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a  -L/bitcoin/depends/x86_64-pc-linux-musl/lib -lQt5FontDatabaseSupport /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Gui.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread -lfontconfig -L/bitcoin/depends/x86_64-pc-linux-musl/lib -lfreetype -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5Gui /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a  -L/bitcoin/depends/x86_64-pc-linux-musl/lib -lQt5FbSupport /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Gui.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5Gui /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a  -L/bitcoin/depends/x86_64-pc-linux-musl/lib -lQt5EventDispatcherSupport /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Gui.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5Gui /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a  -L/bitcoin/depends/x86_64-pc-linux-musl/lib -lQt5EdidSupport /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a  -L/bitcoin/depends/x86_64-pc-linux-musl/lib -lQt5DeviceDiscoverySupport /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a  -L/bitcoin/depends/x86_64-pc-linux-musl/lib -lQt5AccessibilitySupport /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Gui.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5Gui /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a  -L/bitcoin/depends/x86_64-pc-linux-musl/lib -lQt5Network /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a  -L/bitcoin/depends/x86_64-pc-linux-musl/lib -lQt5Widgets /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Gui.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5Gui /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a  -L/bitcoin/depends/x86_64-pc-linux-musl/lib -lQt5Gui /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a  -L/bitcoin/depends/x86_64-pc-linux-musl/lib -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a   -L/bitcoin/depends/x86_64-pc-linux-musl/plugins/platforms  >&5
configure:29434: $? = 0
configure:29435: result: yes
configure:29542: checking whether the linker accepts -lxcb-shm
configure:29561: g++ -m64 -std=c++17 -o conftest -fPIC -pipe -O2  -fno-extended-identifiers -DQT_NETWORK_LIB -I/bitcoin/depends/x86_64-pc-linux-musl/include/QtNetwork -DQT_CORE_LIB -I/bitcoin/depends/x86_64-pc-linux-musl/include/QtCore  -DQT_WIDGETS_LIB -I/bitcoin/depends/x86_64-pc-linux-musl/include/QtWidgets -I/bitcoin/depends/x86_64-pc-linux-musl/include/QtCore -DQT_GUI_LIB -I/bitcoin/depends/x86_64-pc-linux-musl/include/QtGui -DQT_CORE_LIB  -DQT_GUI_LIB -I/bitcoin/depends/x86_64-pc-linux-musl/include/QtGui -DQT_CORE_LIB -I/bitcoin/depends/x86_64-pc-linux-musl/include/QtCore  -DQT_CORE_LIB -I/bitcoin/depends/x86_64-pc-linux-musl/include/QtCore   -I/bitcoin/depends/x86_64-pc-linux-musl/include/  -DHAVE_BUILD_INFO -D__STDC_FORMAT_MACROS -L/bitcoin/depends/x86_64-pc-linux-musl/lib   -lxcb-shm conftest.cpp  >&5
configure:29561: $? = 0
configure:29570: result: yes
configure:29579: checking for QXcbIntegrationPlugin (-lqxcb)
configure:29597: g++ -m64 -std=c++17 -o conftest -fPIC -pipe -O2  -fno-extended-identifiers -DQT_NETWORK_LIB -I/bitcoin/depends/x86_64-pc-linux-musl/include/QtNetwork -DQT_CORE_LIB -I/bitcoin/depends/x86_64-pc-linux-musl/include/QtCore  -DQT_WIDGETS_LIB -I/bitcoin/depends/x86_64-pc-linux-musl/include/QtWidgets -I/bitcoin/depends/x86_64-pc-linux-musl/include/QtCore -DQT_GUI_LIB -I/bitcoin/depends/x86_64-pc-linux-musl/include/QtGui -DQT_CORE_LIB  -DQT_GUI_LIB -I/bitcoin/depends/x86_64-pc-linux-musl/include/QtGui -DQT_CORE_LIB -I/bitcoin/depends/x86_64-pc-linux-musl/include/QtCore  -DQT_CORE_LIB -I/bitcoin/depends/x86_64-pc-linux-musl/include/QtCore   -I/bitcoin/depends/x86_64-pc-linux-musl/include/  -DHAVE_BUILD_INFO -D__STDC_FORMAT_MACROS -L/bitcoin/depends/x86_64-pc-linux-musl/lib  conftest.cpp -lqxcb -lxcb-shm -lqminimal -L/bitcoin/depends/x86_64-pc-linux-musl/lib -lQt5XcbQpa /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5ServiceSupport.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5ThemeSupport.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5FontDatabaseSupport.a -lfontconfig -L/bitcoin/depends/x86_64-pc-linux-musl/lib -lfreetype /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Gui.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5EdidSupport.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5DBus.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread /bitcoin/depends/x86_64-pc-linux-musl/lib/libxcb-static.a -lxcb -lxkbcommon-x11 -lxkbcommon -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5Gui /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5Gui /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5ServiceSupport /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Gui.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5DBus.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5Gui /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5DBus /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5ThemeSupport /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Gui.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5DBus.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5Gui /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5FontDatabaseSupport /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Gui.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread -lfontconfig -L/bitcoin/depends/x86_64-pc-linux-musl/lib -lfreetype -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5Gui /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5EdidSupport /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a  -L/bitcoin/depends/x86_64-pc-linux-musl/lib -lQt5ServiceSupport /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Gui.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5DBus.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5Gui /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5DBus /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a  -L/bitcoin/depends/x86_64-pc-linux-musl/lib -lQt5XcbQpa /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5ServiceSupport.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5ThemeSupport.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5FontDatabaseSupport.a -lfontconfig -L/bitcoin/depends/x86_64-pc-linux-musl/lib -lfreetype /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Gui.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5EdidSupport.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5DBus.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread /bitcoin/depends/x86_64-pc-linux-musl/lib/libxcb-static.a -lxcb -lxkbcommon-x11 -lxkbcommon -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5Gui /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5Gui /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5ServiceSupport /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Gui.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5DBus.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5Gui /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5DBus /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5ThemeSupport /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Gui.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5DBus.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5Gui /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5FontDatabaseSupport /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Gui.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread -lfontconfig -L/bitcoin/depends/x86_64-pc-linux-musl/lib -lfreetype -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5Gui /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5EdidSupport /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a  -L/bitcoin/depends/x86_64-pc-linux-musl/lib -lQt5ThemeSupport /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Gui.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5DBus.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5Gui /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a  -L/bitcoin/depends/x86_64-pc-linux-musl/lib -lQt5FontDatabaseSupport /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Gui.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread -lfontconfig -L/bitcoin/depends/x86_64-pc-linux-musl/lib -lfreetype -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5Gui /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a  -L/bitcoin/depends/x86_64-pc-linux-musl/lib -lQt5FbSupport /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Gui.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5Gui /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a  -L/bitcoin/depends/x86_64-pc-linux-musl/lib -lQt5EventDispatcherSupport /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Gui.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5Gui /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a  -L/bitcoin/depends/x86_64-pc-linux-musl/lib -lQt5EdidSupport /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a  -L/bitcoin/depends/x86_64-pc-linux-musl/lib -lQt5DeviceDiscoverySupport /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a  -L/bitcoin/depends/x86_64-pc-linux-musl/lib -lQt5AccessibilitySupport /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Gui.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5Gui /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a  -L/bitcoin/depends/x86_64-pc-linux-musl/lib -lQt5Network /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a  -L/bitcoin/depends/x86_64-pc-linux-musl/lib -lQt5Widgets /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Gui.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lQt5Gui /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a  -L/bitcoin/depends/x86_64-pc-linux-musl/lib -lQt5Gui /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a -lpthread /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtlibpng.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5Core.a -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtharfbuzz.a -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a  -L/bitcoin/depends/x86_64-pc-linux-musl/lib -lQt5Core -lpthread -lm /bitcoin/depends/x86_64-pc-linux-musl/lib/libqtpcre2.a   -L/bitcoin/depends/x86_64-pc-linux-musl/plugins/platforms  >&5
/usr/lib/gcc/x86_64-alpine-linux-musl/10.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: /bitcoin/depends/x86_64-pc-linux-musl/lib/libQt5XcbQpa.a(qxcbbackingstore.o): in function `QXcbBackingStoreImage::createShmSegment(unsigned long)':
qxcbbackingstore.cpp:(.text._ZN21QXcbBackingStoreImage16createShmSegmentEm+0x5c): undefined reference to `xcb_shm_create_segment'
/usr/lib/gcc/x86_64-alpine-linux-musl/10.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: qxcbbackingstore.cpp:(.text._ZN21QXcbBackingStoreImage16createShmSegmentEm+0x6c): undefined reference to `xcb_shm_create_segment_reply'
/usr/lib/gcc/x86_64-alpine-linux-musl/10.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: qxcbbackingstore.cpp:(.text._ZN21QXcbBackingStoreImage16createShmSegmentEm+0x87): undefined reference to `xcb_shm_create_segment_reply_fds'
collect2: error: ld returned 1 exit status
```

--- a/build-aux/m4/bitcoin_qt.m4
+++ b/build-aux/m4/bitcoin_qt.m4
@@ -150,7 +150,7 @@ AC_DEFUN([BITCOIN_QT_CONFIGURE],[
       AC_DEFINE(QT_QPA_PLATFORM_WINDOWS, 1, [Define this symbol if the qt platform is windows])
     elif test "x$TARGET_OS" = xlinux; then
       dnl workaround for https://bugreports.qt.io/browse/QTBUG-74874
-      AX_CHECK_LINK_FLAG([-lxcb-shm], [QT_LIBS="-lxcb-shm $QT_LIBS"], [AC_MSG_ERROR([could not link against -lxcb-shm])])
+      AX_CHECK_LINK_FLAG([-lxcb-shm], [QT_LIBS="$QT_LIBS -lxcb-shm"], [AC_MSG_ERROR([could not link against -lxcb-shm])])
       _BITCOIN_QT_CHECK_STATIC_PLUGIN([QXcbIntegrationPlugin], [-lqxcb])
       AC_DEFINE(QT_QPA_PLATFORM_XCB, 1, [Define this symbol if the qt platform is xcb])
     elif test "x$TARGET_OS" = xdarwin; then

